pkg/operator/controllers/checker/checker_controller.go

```diff
@@ -37,7 +37,6 @@ func NewReconciler(log *logrus.Entry, maocli maoclient.Interface, arocli aroclie
 
 	if role == operator.RoleMaster {
 		checkers = append(checkers,
-			NewMachineChecker(log, maocli, arocli, role, isLocalDevelopmentMode),
 			NewServicePrincipalChecker(log, maocli, arocli, kubernetescli, role),
 		)
 	}
```

Comment:
`SetupWithManager` needs to be updated here: we no longer want to watch `machinev1beta1.Machine`, it seems

Reply:
Hm. Or do we still need it for some other checker?

Reply:
We could take it or leave it, but I think that if changes are made to the machine objects we would still want to run `Checker` for internet and service principal checks. Let me know what you think.

Reply:
These should be covered by re-queuing. I agree with @m1kola, lets remove them from `checker`

Reply:
Bump.

Reply:
I've removed that watch.